pkg/daemon/manager/manager.go

```diff
@@ -0,0 +1,48 @@
+//go:build windows
+
+package manager
+
+import (
+	"golang.org/x/sys/windows/svc/mgr"
+
+	"github.com/openshift/windows-machine-config-operator/pkg/daemon/winsvc"
+)
+
+type Mgr interface {
```

Comment:
Consider renaming this to `Manager`, that fits within go naming conventions a little better